Dockerfile

```diff
@@ -18,3 +18,18 @@ COPY scripts/pre-stop.sh /usr/local/lib/synthetic-monitoring-agent/pre-stop.sh
 COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 
 ENTRYPOINT ["/usr/local/bin/synthetic-monitoring-agent"]
+
+# third stage with alpine base for better access to chromium
+FROM alpine:3.18 as with-browser
```

Comment:
I think we're mixing two substantially different linux flavors here (alpine and debian) and I'm afraid of unforeseen consequences. Most prominently, the libc implementation is different, which may cause a binary to work in one but not the other. Since the binary is built outside of docker with many many layers of Makefiles, I don't know if it is being built with `CGO_ENABLED=0` (hope/think it is), otherwise it will break on alpine for sure. 

We're also copying the ca-certificates from debian into alpine, which should be okay... but looks strange.

If you have manually tested this works, I wouldn't block the PR for this, but I'd strongly suggest to make a follow-up PR migrating the base image to alpine. We've been looking for an excuse to do this to lower our number of CVEs, so this is an excuse as good as any other.

Reply:
I believe it is being built with `CGO_ENABLED=0` (@mem to confirm if you have a second).

I built and ran this Dockerfile locally and it seems to run the agent binary, though I didn't actually allocate a workload to it so maybe I'll do that and report back.

In either case, using Debian or Alpine, I've noticed that attempting to run headless chromium would result in an error I'm still trying to debug: `Failed to connect to the bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory`. I used `chromiuim`/`chromium-shell` on Debian in that case and got the same issue as `chromium-swiftshader` on Alpine.

I'm still working on understanding that, wondering if you have any insight as you got this running for crocochrome :eyes:

Reply:
If we land on agreeing that this is fine, I would for sure migrate the base to alpine as well.

Reply:
I think that error message is benign, I also get that on crocochrome is my memory serves. Probably just one of the million things chrome is trying to run at startup.

Reply:
I assigned some workloads to this and it seems to run fine on my linux VM -- I will resolve this thread, with the takeaway that we want to migrate the original base to alpine as well!